VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionTopic.java

```diff
@@ -23,12 +25,14 @@
 import java.util.List;
 
 public class DiscussionTopic implements Serializable {
+    public static final String ALL_TOPICS = "All Posts";
     public static final String FOLLOWING_TOPICS = "Posts I'm Following";
     @SerializedName("id") String identifier = "";
     String name = "";
     String threadListUrl = "";
     List<DiscussionTopic> children = new ArrayList<>();
 
+    @Nullable
```

Comment:
What is the purpose of marking the method's return value as `Nullable`? Since the `identifier` variable is initialized to an empty string, the only way it might be null is if it's declared so explicitly in the JSON response.